run.sh

```diff
@@ -2,7 +2,8 @@
 
 
 #Gen3
-export REPO=/project/hsc/gen3repo/rc2v21_0_0_rc1_ssw48
-setup -j -r .
-pipetask run -t lsstdesc.pipe.task.coadd_in_cells.CoaddInCellsTask -b $REPO --input HSC/runs/RC2/v21_0_0_rc1 --output u/$USER/coaddTest -d "skymap='hsc_rings_v1' AND tract=9615 AND patch=45" --register-dataset-types
-butler query-datasets $REPO --collections u/$USER/* coaddObj
+export REPO=/repo/main
+export INPUT_COLLECTION=HSC/runs/RC2/w_2021_30/DM-31182
+setup -k -r .
+pipetask run -t lsstdesc.descwl.coadd.coadd_in_cells.CoaddInCellsTask -b $REPO --input $INPUT_COLLECTION --output u/$USER/coaddTest -d "skymap='hsc_rings_v1' AND tract=9615 AND patch=45 AND band='r'" --register-dataset-types
```

Comment:
I should caution here that the band can be passed via the command line and it will only process one band. However, if you don't specify the band, it will make a coadd by combining exposures from all the bands. I removed the lines that were keeping track of the `band` id since you wanted this to process only one band. Should I keep a check inside the code that ensures that exposures from a single band are used, to prevent any user errors?

Reply:
We should make band a required argument